package.json

```diff
@@ -35,6 +41,12 @@
 	},
 	"homepage": "https://github.com/artykovdaniyar/node.js-mentoring-program#readme",
 	"dependencies": {
-		"nodemon": "^3.1.0"
+		"@types/jest": "^29.5.12",
+		"axios": "^1.6.8",
```

Comment:
Only axios should be a dependency. I also checked the starting repo in case any mistake, it also defines correctly. Just be aware.